toonz/sources/translations/german/colorfx.ts

```diff
@@ -384,8 +384,8 @@
 <context>
     <name>TLinGradFillStyle</name>
     <message>
-        <source>Linear Gradient</source>
-        <translation>Linearer Gradient</translation>
+        <source>Linear Verlauf</source>
```

Comment:
I used terms that were similar to whats in Photoshop, so not sure which is more commonly used